src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -370,8 +376,12 @@ public bool IsDisplayingLabels
             get { return nodeLogic.DisplayLabels; }
             set
             {
-                nodeLogic.DisplayLabels = value;
-                RaisePropertyChanged("IsDisplayingLabels");
+                
+                    DynamoViewModel.ExecuteCommand(
+               new DynamoModel.UpdateModelValueCommand(
```

Comment:
No if for this property?

Reply:
yeah, I'm not sure why that is, I didn't touch it because it didn't exist before and was present for those other properties - I can make the change and see if there are any test failures if you would like.

Reply:
I guess it would be more consistent but not really required. Your call